src/Response/EmptyResponse.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: ycoutu
+ * Date: 13/07/18
+ * Time: 10:51 AM
+ */
+
+namespace ZEROSPAM\Freshbooks\Response;
+
+use ZEROSPAM\Framework\SDK\Response\Api\BaseResponse;
+
+final class EmptyResponse extends BaseResponse
```

Comment:
Should be moved into the SDK-Framework.

I'm going to move it there and update your branch.